docs/releases/3.1.3.txt

```diff
@@ -48,3 +48,14 @@ Bugfixes
 
 * Fixed a regression in Django 3.1.2 that caused incorrect form input layout on
   small screens in the admin change form view (:ticket:`32069`).
+
+* Loosened the ``asgiref`` dependency specifier set in Django 3.1 to allow
+  updating ``asgiref`` to version 3.3.0, and beyond (:ticket:`32128`).
```

Comment:
What about all places where we use `sync_to_async()` without specifying `thread_sensitive`? i.e.

- https://github.com/django/django/blob/755dbf39fcdc491fe9b588358303e259c7750be4/django/core/handlers/base.py#L150-L156

- https://github.com/django/django/blob/755dbf39fcdc491fe9b588358303e259c7750be4/django/core/handlers/exception.py#L40

- https://github.com/django/django/blob/755dbf39fcdc491fe9b588358303e259c7750be4/django/test/client.py#L184

- https://github.com/django/django/blob/755dbf39fcdc491fe9b588358303e259c7750be4/django/test/client.py#L199-L208

- https://github.com/django/django/blob/755dbf39fcdc491fe9b588358303e259c7750be4/django/contrib/staticfiles/handlers.py#L57-L61


Should we change them? I think yes, to keep the same behavior for both `asgiref` 3.2 and 3.3.

I would also change release notes to:

```
* Added support for ``asgiref`` 3.3 (:ticket:`32128`).
```

without any note. IMO we don't need to document changes in requirements.